schema/artist/meta.js

```diff
@@ -0,0 +1,42 @@
+import { slugs } from './maps/artist_title_slugs';
+import { stripTags } from '../../lib/helpers';
+import {
+  GraphQLString,
+  GraphQLObjectType,
+} from 'graphql';
+
+
+const countDisplay = (artist) => {
+  const count = artist.published_artworks_count;
+
+  if (count < 10) { return ''; }
+  if (count < 100) { return `${Math.floor(count / 10) * 10}+ `; }
+  if (count < 1000) { return `${Math.floor(count / 100) * 100}+ `; }
+  if (count < 10000) { return `${Math.floor(count / 1000) * 1000}+ `; }
+  return `${Math.floor(count / 10000) * 10000}+ `;
+};
+
+const metaName = (artist) => {
```

Comment:
This comes from force's implementation of artist title tags. Seems prudent to keep in the weird defensiveness it has.